cmake_configure.cmake

```diff
@@ -188,25 +197,36 @@ endforeach()
 
 function(openrw_target_apply_options)
     set(IWYU_MAPPING "${PROJECT_SOURCE_DIR}/openrw_iwyu.imp")
-    cmake_parse_arguments("ORW" "INSTALL;INSTALL_PDB;COVERAGE" "TARGET" "COVERAGE_EXCEPT" ${ARGN})
-    if(CHECK_IWYU)
-        iwyu_check(TARGET "${ORW_TARGET}"
-            EXTRA_OPTS
-                "--mapping_file=${IWYU_MAPPING}"
-        )
-    endif()
+    cmake_parse_arguments("ORW" "CORE;INSTALL;INSTALL_PDB;COVERAGE" "TARGET" "COVERAGE_EXCEPT" ${ARGN})
 
     if(TEST_COVERAGE AND ORW_COVERAGE)
         coverage_add_target("${ORW_TARGET}" EXCEPT ${ORW_COVERAGE_EXCEPT})
     endif()
 
-    if(CHECK_CLANGTIDY)
-        clang_tidy_check_target(
-            TARGET "${ORW_TARGET}"
-            FORMAT_STYLE "file"
-            FIX "${CHECK_CLANGTIDY_FIX}"
-            CHECK_ALL
-        )
+    if(ORW_CORE)
+        if(CHECK_IWYU)
+            iwyu_check(TARGET "${ORW_TARGET}"
+                EXTRA_OPTS
+                    "--mapping_file=${IWYU_MAPPING}"
+            )
+        endif()
+
+        set_target_properties("${ORW_TARGET}"
+            PROPERTIES
+                CXX_EXTENSIONS OFF
+                CXX_STANDARD 17
+                C_EXTENSIONS OFF
+                C_STANDARD 99
```

Comment:
c11?

Reply:
Right now, the only C file are the gl bindings in rwcore.
This change will mean that openrw will be a c++17/c11 project.